koku/api/report/ocp_aws/ocp_aws_query_handler.py

```diff
@@ -38,18 +38,15 @@ def __init__(self, query_parameters, url_data,
             tenant    (String): the tenant to use to access CUR data
             kwargs    (Dict): A dictionary for internal query alteration based on path
         """
-        self._provider = 'OCP_AWS'
-        kwargs['provider'] = self._provider
-        super().__init__(query_parameters, url_data,
-                         tenant, **kwargs)
-        self.update_report_map()
-
-    def update_report_map(self):
```

Comment:
This is a couple of simplifications. Realistically, we shouldn't be doing string-checking like this. We should be using `isinstance` instead. I'll make another pass in the future to clean this up further.